tests/baselines/reference/functionsWithImplicitReturnTypeAssignableToUndefinedStrictNullChecks.js

```diff
@@ -0,0 +1,27 @@
+//// [functionsWithImplicitReturnTypeAssignableToUndefinedStrictNullChecks.ts]

```

Comment:
Note that old baselines are not deleted by default; you'll want to remove these baselines that no longer map to any test.

Reply:
Ah, the more I learn. Hopefully the third time's the charm.